tests/test_extract.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from etl.extract import extract_csv_data, extract_fund_info
+
+
+def test_extract_fund_info() -> None:
+    sample = "Fund Whitestone.28-02-2023 - details"
+    name, date = extract_fund_info(sample)
+    assert name == "Whitestone"
+    assert date == "02/28/2023"
+
+
+def test_extract_csv_data(tmp_path: Path) -> None:
+    csv_content = "price,market value,quantity\n100.5,1000,10"
```

Comment:
Would also be ok to commit some small CSV files in a `/tests/data` directory instead of writing to `tmp_path`.